data/korath/korath ships.txt

```diff
@@ -296,6 +313,219 @@ ship "Korath Raider" "Korath Raider (Crippled)"
 	turret "Korath Warder"
 
 
+ship "Korath Raider Mark XLII"
+	sprite "ship/raiderii"
+	thumbnail "thumbnail/raiderii"
+	attributes
+		category "Medium Warship"
+		"cost" 22395000
+		"shields" 27500
+		"hull" 11000
+		"required crew" 157
+		"bunks" 250
```

Comment:
with the addition of the gun port shuttle thing below, shouldn't we have a bit more bunks? Maybe just 5 extra, but I feel like there should be some more.
Also maybe add 500 more shields?

Reply:
I think the bunk count is fine. And shields are already buffed over the extant Raider.

Reply:
The idea behind not buffing bunks is although the Raider Mark XLII is a newer model of the Raider, it's not an increased sized raider. It has more functionality, more modules, but not more room for crew. Think of adding extra arsenal to WWII plane- additional gun pods and cannons under the wings, but still the same frame and pilot count.

Shields were already buffed by 500, armor by 1000. It makes sense for these ratios based on the visual update to the design.